opcodes/riscv-dis.c

```diff
@@ -277,12 +277,14 @@ print_insn_args (const char *d, insn_t l, bfd_vma pc, disassemble_info *info)
 	  if (d[1] == '1')
 	    {
 	      info->target = (EXTRACT_ITYPE_IMM (l)<<1) + pc; ++d;
+	      print (info->stream, "0x");
 	      (*info->print_address_func) (info->target, info);
 	      break;
-            }
+	    }
 	  else if (d[1] == '2')
 	    {
 	      info->target = (EXTRACT_CV_HWLP_UIMM5 (l)<<1) + pc; ++d;
+	      print (info->stream, "0x");
```

Comment:
@jessicamills Can you explain why you need to print this here? I note that for `c.beqz` which uses the `Cp` operand type, doesn't need this, because objdump prints it (see line 234 of this file)